src/java.base/share/classes/java/lang/reflect/Executable.java

```diff
@@ -257,13 +257,20 @@ public int getParameterCount() {
     }
 
     /**
-     * Returns an array of {@code Type} objects that represent the formal
-     * parameter types, in declaration order, of the executable represented by
-     * this object. Returns an array of length 0 if the
-     * underlying executable takes no parameters.
-     * Note that the constructors of some inner classes
-     * may have an implicitly declared parameter in addition to
-     * explicitly declared ones.
+     * Returns an array of {@code Type} objects that represent the
+     * formal parameter types, in declaration order, of the executable
+     * represented by this object. Returns an array of length 0 if the
```

Comment:
Missing subject of the sentence?  "This method returns"...
Or "An array of length 0 is returned"...